tests/conftest.py

```diff
@@ -0,0 +1,56 @@
+import os
+import os.path
+
+import pytest
+
+from pytest_bdd import given
+from pytest_bdd import parsers
+from pytest_bdd import then
+from pytest_bdd import when
+
+from utils.helper import run_ansible_playbook
+
+
+@pytest.fixture(scope="session")
+def inventory():
+    inventory_file = os.environ.get('ANSIBLE_INVENTORY')
+    assert os.path.exists(inventory_file)
+    if not os.path.exists(inventory_file):
+        pytest.fail(
+            "The path specified by ANSIBLE_INVENTORY environment variable "
+            "does not exist: '{0}'".format(inventory_file)
+        )
+    return inventory_file
+
+
+@pytest.fixture(scope="session")
+def kubeconfig(inventory):
```

Comment:
I'd really prefer `KUBECONFIG` to be configurable (and required to be set), not magical and relative to `ANSIBLE_INVENTORY`.

Reply:
I think making a fallback towards the default path where kubespray puts the admin.conf makes sense if not specified, but having to specify KUBECONFIG seems too much for me.
Having a way to override it so we can execute the test towards a platform where the initial test failed is the only thing I would care about to be honest.